packages/web-react/src/components/Accordion/useAccordionStyleProps.ts

```diff
@@ -0,0 +1,39 @@
+import { useClassNamePrefix } from '../../hooks';
+
+export interface AccordionStyleReturn {
```

Comment:
In many components there is used convention same as`<Component>Styles` I suggest do it the same.
```suggestion
export interface AccordionStyles {
```

Reply:
If it is the props that the hook returns, then it should be named with the Return suffix, if I have input props, they will be named like this. Just like `useAccordionAriaProps`

Reply:
We should stay with consistent naming. If we want to rename all the return types to use the suffix `Return`, let's discussed it elsewhere.

I also think that this is a good topic to discuss, e. g. we should use some style guide (for example https://basarat.gitbook.io/typescript/styleguide) for Typescript names to remain consistent.

Reply:
MUI also uses the `Return` suffix